devops/scripts/benchmarks/benches/compute.py

```diff
@@ -432,13 +361,84 @@ def run(
                     command=command,
                     env=env_vars,
                     unit=unit,
-                    git_url=self.bench.git_url(),
-                    git_hash=self.bench.git_hash(),
+                    git_url=self._bench.git_url(),
+                    git_hash=self._bench.git_hash(),
                 )
             )
         return ret
 
-    def parse_output(self, output: str) -> list[tuple[float, float]]:
+    def explicit_group(self):
+        return ""
+
+    def enabled(self) -> bool:
+        # SYCL is required for all benchmarks
+        if options.sycl is None:
+            return False
+
+        # HIP adapter is not supported
+        if options.ur_adapter == "hip":
+            return False
+
+        # Check if the specific runtime is enabled (or no specific runtime required)
+        return self._runtime is None or self._runtime in self.__enabled_runtimes()
+
+    def description(self) -> str:
+        return ""
+
```

Comment:
perhaps also, describe this shortly in our contrib guide...?

Reply:
Added a guideline to contrib guide